modules/op-running-pipeline-and-task-run-pods-with-privileged-security-context.adoc

```diff
@@ -0,0 +1,69 @@
+[id='op-running-pipeline-and-task-run-pods-with-privileged-security-context']
+= Running pipeline run and task run pods with privileged security context
+:context: op-running-pipeline-and-task-run-pods-with-privileged-security-context
+
+toc::[]
+
+.Procedure
+
+To run a pod (resulting from pipeline run or task run) with `privileged` security context, do the following modifications:
+
+* Configure the associated user account or service account to have an explicit SCC. You can perform the configuration using any of the following methods:
+** Use the following OpenShift command:
++
+[source, terminal]
+----
+$ oc adm policy add-scc-to-user <scc-name> -z <service-account-name>
+----
+** Manually modifying the YAML for `RoleBinding` and `Role` or `ClusterRole`:
+
++
+.Example `RoleBinding` object
+[source,yaml,subs="attributes+"]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  name: service-account-name <1>
+  namespace: default
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: ClusterRole
+  name: pipelines-scc-clusterrole <2>
+subjects:
+- kind: ServiceAccount
+  name: pipeline
+  namespace: default
+
+----
+<1> Substitute with the same service account name as used in the `oc` command above.
+<2> Substitute with appropriate cluster role based on the role binding you use.
+
++
+.Example `ClusterRole` object
+[source,yaml,subs="attributes+"]
+----
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: pipelines-scc-clusterrole <1>
+rules:
+- apiGroups:
+  - security.openshift.io
+  resourceNames:
+  - nonroot
+  resources:
+  - securitycontextconstraints
+  verbs:
+  - use
+----
+<1> Substitute with appropriate cluster role based on the role binding you use.
+
++
+[NOTE]
+====
+As a best practice, create a copy of the default YAML files and make changes in the duplicate file.
+====
++
+
+* If you do not use the `vfs` storage driver, configure the service account associated with the task run or the pipeline run to have `privileged` SCC, and set the security context as `privileged: true`.
```

Comment:
have a `privileged` SCC

Reply:
Removed the back ticks surrounding `privileged` and made it a generic word, because that makes more sense, especially because soon after we are asking the user to set the security context to `privileged: true` (which is the actual actionable part).